src/react-components/src/components/03-organisms/SearchPromo/SearchPromo.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+
+import SearchPromo from "./SearchPromo";
+import { action } from "@storybook/addon-actions";
+import { withKnobs, text } from "@storybook/addon-knobs";
+
+
+export default {
+  title: "SearchPromo",
+  component: SearchPromo,
+};
+
+export const searchPromo = () => <SearchPromo headingText="Search the World's Research Collections"
+  titleId="title"
+  advancedSearchMessage={<p>click here for <a href="#apple-pie">stuff</a></p>}
+  searchButtonId="searchButtonId"
+  searchBarId="searchBarId" dropdownId="dropdownId"
+  searchDropdownOptions={["hi there", "bye there"]}
+  searchSubmitHandler={action("searched")}
+  selectChangeHandler={action("changed")}
+  selectBlurHandler={action("selectChanged")} />;
+export const searchPromoWithError = () => <SearchPromo headingText="Search the World's Research Collections"
```

Comment:
The dropdown is off when there's an error:
![Screen Shot 2019-12-16 at 3 11 17 PM](https://user-images.githubusercontent.com/1280564/70939471-583d4080-2016-11ea-936e-f3460f8b0c74.png)